### DISCUSSION TITLE
Prevent `fastn-lsp` from auto selection of first suggestion on enter key press

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-stack/fastn#809] [Ideas & RFCs]
Modify `fastn-lsp` to prevent automatic selection of the first suggestion on enter key press.

The suggestions list of the `fastn-lsp` should not select the first recommendation as when we hit `enter` the text gets modified by the first recommendation word. The options from the suggestion list should only be highlighted using the navigational arrow keys.

For example:
![image](https://user-images.githubusercontent.com/118595104/229417966-b50f3809-133d-4ef7-acfb-9e9a54c7310c.png)

User may wish to write only `menu` and wants to go to the following line by hitting `enter` but the LSP suggestion list highlights `menu-icon` so the text gets updated to `menu-list` when we hit `enter`.

**Comment by amitu** [comment]
There is no fastn-lsp yet. This is default behaviour of the editor you are using.
